scripts/init.js

```diff
@@ -18,6 +18,7 @@
       type: "text",
       name: "name",
       message: "What is the name of your test app?",
+      initial: "testapp",
```

Comment:
Since this is the app name, I think we should capitalize it:
```suggestion
      initial: "TestApp",
```